app/helpers/basic_auth_helper.rb

```diff
@@ -0,0 +1,13 @@
+module BasicAuthHelper
+  def blanket_basic_auth
```

Comment:
`blanket` is a verb as well as a noun, and it's nice to have `before_actions` be verbs. How would you feel about `require_basic_auth` or sth? Or is the fact that it is "blanket" important for callers to know?

Reply:
Will use :require_basic_auth_for_ui